src/shoji.py

```diff
@@ -0,0 +1,23 @@
+"""Shoji helper functions."""
+
+# These have been extracted from server/tests/controllers/base.py
```

Comment:
```suggestion

```
Given that Pycrunch is a public library, it's not necessary to hold references to our internal codebase files.

Reply:
It also occurs to me that this code needs added inside the existing shoji module, not here as a standalone. 

So both of these should now be addressed. 

(You alluded to that fact before, I just picked up on that. I may be slow, but I tell myself you don't need big brains to have fun)